homeassistant/components/device_tracker/icloud.py

```diff
@@ -44,12 +45,24 @@ def setup_scanner(hass, config, see):
         _LOGGER.exception('Error logging into iCloud Service: %s', error)
         return False
 
-    def keep_alive(now):
-        """ Keeps authenticating iCloud connection. """
+    def lost_iphone(hostname):
+        """ Calls the lost iphone function if the device is found """
         api.authenticate()
-        _LOGGER.info("Authenticate against iCloud")
+        for device in api.devices:
+            status = device.status()
+            _LOGGER.info('lost_iphone: check device %s', status['name'])
+            if hostname == status['name']:
+                _LOGGER.info('lost_iphone: device %s found, playing sound',
+                             status['name'])
+                device.play_sound()
 
-    track_utc_time_change(hass, keep_alive, second=0)
```

Comment:
the keep alive function isn't necessary anymore, you only get a mail when you login the first time